file.go

```diff
@@ -469,26 +469,30 @@ type filePages struct {
 }
 
 func (r *filePages) readPage() (*filePage, error) {
-	h := &r.page.header
-	h.Type = 0
-	h.UncompressedPageSize = 0
-	h.CompressedPageSize = 0
-	h.CRC = 0
+	r.page.header = format.PageHeader{}
 
-	if h.DataPageHeader != nil {
-		*h.DataPageHeader = format.DataPageHeader{}
-	}
-	if h.IndexPageHeader != nil {
-		h.IndexPageHeader = nil
-	}
-	if h.DictionaryPageHeader != nil {
-		*h.DictionaryPageHeader = format.DictionaryPageHeader{}
-	}
-	if h.DataPageHeaderV2 != nil {
-		*h.DataPageHeaderV2 = format.DataPageHeaderV2{}
-	}
+	/*
```

Comment:
Purposeful or not

Reply:
Yes, this code was not used (see comment above), it was confusing to keep it around.